creator-node/src/routes/files.js

```diff
@@ -115,7 +115,7 @@ const getCID = async (req, res) => {
 
   try {
     // Add a rehydration task to the queue to be processed in the background
-    RehydrateIpfsQueue.addRehydrateIpfsFromFsIfNecessaryTask(CID, queryResults.storagePath, { logContext: req.logContext })
+    await RehydrateIpfsQueue.addRehydrateIpfsFromFsIfNecessaryTask(CID, queryResults.storagePath, { logContext: req.logContext })
```

Comment:
These awaits aren't bugs. They were intentional before to make adding to queue happen in background, but i feel like we should ensure it got added to the queue